STAT6250-01_w18-team-1_project1_data_analysis_by_AA.sas

```diff
@@ -122,7 +129,7 @@ proc means
         Victim_Age
 	;
 	output
-		out=homicide_analytic_file_temp
+		out=homicide_analytic_file_infant
```

Comment:
You can probably delete this... i didn't see the homicide_analytic_file_infant data set being used elsewhere in code.